joblib/testing.py

```diff
@@ -43,13 +43,12 @@ def assert_raises_regex(expected_exception, expected_regexp,
             raise AssertionError("Should have raised %r" %
                                  expected_exception(expected_regexp))
 
-try:
-    SkipTest = unittest.case.SkipTest
-except AttributeError:
-    # Python <= 2.6, we still need nose here
-    SkipTest = nose.SkipTest
+SkipTest = pytest.skip
```

Comment:
I just had a look at the documentation and I agree with you. Modified :white_check_mark: